pkg/controller/datavolume/controller-base.go

```diff
@@ -796,10 +810,27 @@ func (r ReconcilerBase) updateStatus(req reconcile.Request, phaseSync *statusPha
 				if err != nil {
 					return reconcile.Result{}, err
 				}
-				if shouldBeMarkedWaitForFirstConsumer {
```

Comment:
I'm wondering if the `shouldBeMarkedWaitForFirstConsumer` should be updated.  I'm not sure it should return true if the nodeName annotation has been added (at least for populators).  We should be able to start reporting the phases of pvc' at that point.  I don't think they'll get reported at all otherwise, well until the target is bound which is when succeeded

Reply:
ok will do something else